articles/arm-x86/index.md

```diff
@@ -0,0 +1,85 @@
+
+Another benefit of ARM is that it's a semi-open architecture. [Very few companies manufacture x86 processors](https://en.wikipedia.org/wiki/List_of_x86_manufacturers), because it's owned by Intel. Intel doesn't want too much competition. ARM doesn't actually manufacture their own CPUs. [They license the design out to other companies](https://developer.arm.com/support/licensing) who want to make their own CPUs. [Apple is one of them](https://en.wikipedia.org/wiki/Apple_Silicon). Apple is also able to customize their chips to work better on their platforms. That's presumably why [iPhones perform very well in benchmarks](https://benchmarks.ul.com/compare/best-smartphones?amount=200&sortBy=PERFORMANCE&reverseOrder=true&osFilter=ANDROID,IOS&test=SLING_SHOT_ES_30_UNLIMITED&deviceFilter=PHONE&displaySize=3.0,15.0).
+
+ARM is commonly used in smartphones. It's smaller, more energy-efficient, and creates less heat.  This makes it perfect for mobile devices. Laptops don't seem to be an exception to this. ARM would Apple to make their MacBooks cooler. They've historically had problems with this in [2020](https://www.macworld.co.uk/news/mac/2020-macbook-air-problems-3788127/), [2018](https://www.zdnet.com/article/apple-patches-2018-macbook-pro-to-address-throttled-performance-and-overheating/), [2015](https://www.theverge.com/2019/6/20/18693136/apple-recall-2015-15-inch-macbook-pro-battery-overheat-fire-risk-safety), and [older](https://discussions.apple.com/thread/5815813). Extending the battery life also sounds very nice. [There's also a possibility of Apple's laptops getting smaller](https://www.forbes.com/sites/ewanspence/2020/07/20/apple-macos-bigsur-macbook-pro-arm-intel-advantages-danger/#1354fad339f7). Although, consumers most likely won't care about that as much. Laptops can already be [very light](https://www.lg.com/us/laptops/lg-14z90n-u.aas7u1-ultra-slim-laptop).
+
+## Drawbacks of ARM
+
+There is an issue of compatibility here though. Programs will need to be completely rewritten to run on Apple's newer machines. This isn't too bad, since most compilers can target ARM just fine. Assembly developers are probably having a panic attack right now. Apple is also hoping that [Rosetta 2](https://www.theverge.com/21304182/apple-arm-mac-rosetta-2-emulation-app-converter-explainer) can run all the x86 applications. Rosetta 1 was known for being much slower than running on original hardware. This is by necessity. Rosetta needs to translate the x86 instructions into ARM instructions in real-time. To be fair, Java converts from [bytecode to everything else](https://www.javatpoint.com/java-bytecode), and it seems to work [fine-ish](https://benchmarksgame-team.pages.debian.net/benchmarksgame/fastest/java.html). Still, it seems hard to imagine being able to play games from the 2010's if Microsoft also decides to switch to ARM.
```

Comment:
First sentence unnecessary/too vague. Explain the concept of backwards compatibility, and remove evaluative/judgmental phrases like "this is cool"/"this isn't too bad" and speculations like "developers are having a panic attack". Keep it formal and informative.

Also, when you write "Apple is hoping that..." this falls into the realm of speculation. Rather, only refer to what Apple has publicly stated, and corroborate any inferences.